projects/storefrontlib/src/cms-components/user/register/register.component.spec.ts

```diff
@@ -257,11 +331,11 @@ describe('RegisterComponent', () => {
       });
     });
 
-    it('should go to redirect url after registration and remove error messages', () => {
-      expect(authRedirectService.redirect).toHaveBeenCalled();
-      expect(globalMessageService.remove).toHaveBeenCalledWith(
-        GlobalMessageType.MSG_TYPE_ERROR
-      );
+    it('should redirect to login page and show message', () => {
```

Comment:
You could leave old test and add new one for new behavior. Manipulating feature flag levels will help with maintaining 2 versions.